sos/report/plugins/ssacli.py

```diff
@@ -0,0 +1,50 @@
+# ssacli.py
+
+from sos.report.plugins import Plugin, RedHatPlugin
+import re
+
+
+class Ssacli(Plugin, RedHatPlugin):
+    """
+    This plugin will capture information about the Smart Storage Array
+    controller(s) configuration on the system.
+    """
+    short_desc = 'HPE Smart Array devices'
+    plugin_name = 'ssacli'
+    profiles = ('system', 'storage', 'hardware')
+    files = ('/opt/smartstorageadmin/ssacli/bin/ssacli',)
+    packages = ('ssacli',)
+
+    def setup(self):
+        cmd = '/opt/smartstorageadmin/ssacli/bin/ssacli'
```

Comment:
Is this really not typically installed to a user's PATH? Having a bunch of `opt_smartstorage_admin_ssacli_bin_ssacli_*` prefixed outputs isn't exactly easy on the eyes. Perhaps leveraging `add_cmd_output()`'s `runat` parameter could help here? E.G. have `cmd` be the stock `ssacli` and set `runat` to the `/opt/....` path?

Reply:
It is in current versions of the RPM. My pedantic memory recalls when it was not, so I tend to use the full path excessively. Given it is now part of `/usr/sbin/` for a few years I'd be happy to drop this entirely and rely on the defined PATH.